src/qudi/hardware/interfuse/ni_scanning_probe_interfuse.py

```diff
@@ -113,7 +116,6 @@ def __init__(self, config, **kwargs):
         self._default_timer_interval_ms = -1
         self._interval_time_stamp = None
 
-        self.__read_pos = -1
         self.__scan_stopped = False
```

Comment:
I think we can drop __scan_stopped. It's only written, never checked in this module.

Reply:
Yep, you're right. Might use it though, when I will rework the "RawData" thingy again.

Reply:
It was actually is checked. Yet "wrapped" under _check_scan_end_reached. But maybe you ment getting rid of that "wrapped" structure, which I did now.

Reply:
In line 520 and 523, it's the only time that __scan_stopped is set True. Thus, all other code could simply call _check_scan_end_reached() to test this condition. I see no need to keep the class variable there then.

Reply:
I did completely remove _check_scan_end_reached and __scan_stopped so far and used a property of the RawDataContainer. Since, this will be gone at some point I did include the _check_scan_end_reached again in 1c522be yet using the property .is_full() so __scan_stopped is gone as requested.